source/gold/src/gold/domain/schemas/gold_measurements.py

```diff
@@ -0,0 +1,25 @@
+﻿from pyspark.sql.types import (
+    DecimalType,
+    StringType,
+    StructField,
+    StructType,
+    TimestampType,
+)
+
+from gold.domain.constants.column_names.gold_measurements_column_names import (
+    GoldMeasurementsColumnNames,
+)
+
+measurements_gold_schema = StructType(
```

Comment:
```suggestion
gold_measurements_schema = StructType(
```

Shouldn't we have layer first? We have done this other places, just to align